lib/json.ps1

```diff
@@ -148,47 +148,58 @@ function normalize_values([psobject] $json) {
     # Iterate Through Manifest Properties
     $json.PSObject.Properties | ForEach-Object {
 
+        $value = $_.Value
         # Recursively edit psobjects
         # If the values is psobjects, its not normalized
         # For Example if manfiest have architecture and it's architecture have array with single value it's not formated
         # @see: Without recursion: https://i.imgur.com/pHOf9f2.png
         # @see: With recurseion: https://i.imgur.com/QVSLaiV.png
-        if ($_.Value -is [System.Management.Automation.PSCustomObject]) {
-            $_.Value = normalize_values $_.Value
+        if ($value -is [System.Management.Automation.PSCustomObject]) {
+            $vallue = normalize_values $value
```

Comment:
`$vallue`? 😄

Reply:
Line 151. I extracted `$_.Value` into variable for less typing.